### PR TITLE
Update link in PROCEDURES.md

### LINKED ISSUE #274: Wrong link in the release PROCEDURES.md
Under https://github.com/knative/release/blob/main/PROCEDURES.md/#permissions-for-release-leads

This link is invalid:
[knative/community/main/peribolos/knative-extensions.yaml#Knative Release Leads](https://github.com/knative/community/blob/e635686d46366906af861c409978c2c55990a10e/peribolos/knative-extensions.yaml#L739)

The correct link should be https://github.com/knative/community/blob/e635686d46366906af861c409978c2c55990a10e/peribolos/knative-sandbox.yaml

But as we can see file name is still knative-sandbox.yaml. Maybe we should consider updating the name if it is not complicated.

### PR DESCRIPTION
**Pull request by prashantrewar**
# Changes

-
-
-

/kind <kind>

Fixes #274 

**Release Note**

```release-note
NONE
```

**Docs**

```docs

```

### CODE REVIEW

**Comment by Leo6Leo** [comment]
/lgtm
Thanks for your contribution @prashantrewar ! This looks good to me
/cc @upodroid @pierDipi

**Review by upodroid — CHANGES_REQUESTED** [review]
Please update the commit hash to a more recent one where the file exists.

**Comment by prashantrewar** [comment]
> Please update the commit hash to a more recent one where the file exists.

Done! @upodroid

**Comment by krsna-m** [comment]
Newer hash should point at the knative-extensions not the old sandbox repo

**Review by ReToCode — DISMISSED** [review]
@prashantrewar would you updating the PR to use `knative-extensions` instead of `knative-sandbox`?

File: PROCEDURES.md [hunk 1]
```diff
@@ -247,7 +247,7 @@ During a release, the release leads for that cycle need to be given all the perm
 Check if the new leads are included in/removed from these two files in the `Knative Release Leads` section:
 
 - [knative/community/main/peribolos/knative.yaml#Knative Release Leads](https://github.com/knative/community/blob/e635686d46366906af861c409978c2c55990a10e/peribolos/knative.yaml#L878)
-- [knative/community/main/peribolos/knative-extensions.yaml#Knative Release Leads](https://github.com/knative/community/blob/e635686d46366906af861c409978c2c55990a10e/peribolos/knative-extensions.yaml#L739)
+- [knative/community/main/peribolos/knative-sandbox.yaml#Knative Release Leads](https://github.com/knative/community/blob/b3ebd3d87db759612256acad392722cf977fed53/peribolos/knative-sandbox.yaml)
```
Comment (skonto) [inline PROCEDURES.md:250]: Do we want to define the line the file? The current link for main I see is: https://github.com/knative/community/blob/d9633c5bc6b2557e359f5402f998d20e3a51502a/peribolos/knative-extensions.yaml#L747.

Comment (prashantrewar) [reply PROCEDURES.md]: I updated the changes.

**Comment by prashantrewar** [comment]
hey @upodroid and @pierDipi, I updated the changes. Could you please take a look and suggest changes if required?

**Review by upodroid — APPROVED** [review]
Thanks

**Pull request merged**